alacritty_terminal/src/term/mod.rs

```diff
@@ -376,6 +310,12 @@ impl RenderableCell {
         }
     }
 
+    fn is_empty(&self) -> bool {
+        self.bg_alpha == 0.
+            && !self.flags.intersects(Flags::UNDERLINE | Flags::STRIKEOUT)
+            && self.inner == RenderableCellContent::Chars([' '; cell::MAX_ZEROWIDTH_CHARS + 1])
+    }
```

Comment:
I'd assume that it was a performance optimization, since this check is a bit simpler to some point?

Reply:
It's just that we don't have access to the `selected` information anymore since that's now in the RenderableCells creation method. So I've opted to instead just decide based on the resulting cell rather than the input cell.

It is true though that this should be more efficient since it skips checks that aren't useful.